x-pack/plugins/ml/public/application/datavisualizer/index_based/components/field_data_card/top_values/top_values.tsx

```diff
@@ -43,32 +45,39 @@ export const TopValues: FC<Props> = ({ stats, fieldFormat, barColor }) => {
     isTopValuesSampled,
   } = stats;
   const progressBarMax = isTopValuesSampled === true ? topValuesSampleSize : count;
-
   return (
     <div data-test-subj="mlFieldDataCardTopValues">
-      {topValues.map((value: any) => (
-        <EuiFlexGroup gutterSize="xs" alignItems="center" key={value.key}>
-          <EuiFlexItem grow={false} style={{ width: 100 }} className="eui-textTruncate">
-            <EuiToolTip content={kibanaFieldFormat(value.key, fieldFormat)} position="right">
-              <EuiText size="xs" textAlign="right" color="subdued">
-                {kibanaFieldFormat(value.key, fieldFormat)}
+      {Array.isArray(topValues) &&
+        topValues.map((value: any) => (
+          <EuiFlexGroup gutterSize="xs" alignItems="center" key={value.key}>
+            <EuiFlexItem
+              grow={false}
+              className={classNames(
+                'eui-textTruncate',
+                'mlTopValuesLabelContainer',
+                compressed === true ? 'small' : 'large'
```

Comment:
With the SASS changes suggested in the other comment this should become e.g. `mlTopValuesLabelContainer--${compressed === true ? 'small' : 'large'}`

Reply:
Updated here https://github.com/elastic/kibana/pull/85726/commits/0dcdf2efbdbfdbd6a0259f38cc37bac7942cfae7